package.json

```diff
@@ -79,6 +85,10 @@
     "stylelint-prettier": "^1.1.2",
     "stylelint-scss": "^3.13.0",
     "tcp-port-used": "^1.0.1",
+    "testcafe": "^1.8.4",
+    "testcafe-angular-selectors": "^0.4.1",
+    "testcafe-ovh-page-models": "blaryjp/testcafe-ovh-page-models",
```

Comment:
of course, this will be released on ovh organisation

Reply:
Why do you not work on `ovh` though ? 🤔

Reply:
because I don't want to publish this repository until that I'm sure that this pull request works

Reply:
🤔 

and can't you instead create a private repo for `ovh` ?

Reply:
indeed! I didn't remember that Github opened for free the private repos :D ! thx!